packages/open-payments/src/client/quote.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpMethod } from 'openapi'
+import { RouteDeps } from '.'
+import { getRSPath, Quote } from '../types'
+import { get } from './requests'
+
+interface GetArgs {
+  url: string
+}
+
+export interface QuoteRoutes {
+  get(args: GetArgs): Promise<Quote>
+}
+
+export const createQuoteRoutes = (deps: RouteDeps): QuoteRoutes => {
+  const { axiosInstance, openApi, logger } = deps
+
+  const getQuoteValidator = openApi.createResponseValidator<Quote>({
+    path: getRSPath('/quotes/{id}'),
+    method: HttpMethod.GET
+  })
+
+  return {
+    get: (args: GetArgs) =>
+      get({ axiosInstance, logger }, args, getQuoteValidator)
```

Comment:
@wilsonianb 

The only validation I can think of when fetching a quote is making sure that the `sendAmount.value` or `receiveAmount.value` are positive integers... is this worth adding here?

Reply:
I think the `format: uint64` should cover that.
https://github.com/interledger/open-payments/blob/main/openapi/schemas.yaml#L116-L123